app/client/main.py

```diff
@@ -1,83 +1,234 @@
 import requests, os, time
+from dotenv import load_dotenv
+load_dotenv()
 
-# Flow
-# 1. Client registers with their google account
-# 1.1 Client gets redirected to google
-# 1.2 Client gets an access token
-# 1.3. Client sends request to server with access token
-# 2. Client checks their contexts
-# 3. Client creates a context
-# 4. Client creates an item
-# 5. Client checks their items
-# 6. Client updates an item
-
-ACCESS_TOKEN = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJlZXJzb295NjFAZ21haWwuY29tIiwiZXhwIjoxNjkyNjI1NzUyfQ.h5In0Z87O__a5ITQPCWTot6_2ngkzRrzwEyd_o_WzkA'
+ACCESS_TOKEN = os.environ.get('ADMIN_ACCESS_TOKEN')
```

Comment:
It is usually a good idea to use the same variable name for the application and the container.

Reply:
Done!